builtin/rebase.c

```diff
@@ -817,6 +823,19 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 		}
 	}
 
+	if (gpg_sign) {
+		struct strbuf buf2 = STRBUF_INIT;
+
+		strbuf_reset(&buf);
+		strbuf_reset(&buf2);
+		strbuf_addf(&buf2, "--%s", gpg_sign);
+		strbuf_addstr(&buf, "-S");
+		sq_quote_buf(&buf, buf2.buf);
```

Comment:
`add_var()` already performs the necessary sq-quoting. Please keep that in mind.

Therefore, it is unnecessary here.